server/model/monitor.js

```diff
@@ -1526,6 +1543,23 @@ class Monitor extends BeanModel {
         ]);
     }
 
+    /**
+     * Gets Parent name
+     * @returns {Promise<string>} Name of this monitor's parent
+     */
+    async getParentName() {
+        let name = "";
+
+        if (this.parent === null) {
+            return name;
+        }
+
+        let parent = await Monitor.getParent(this.id);
+        name = `${parent.name}`;
+
+        return name;
```

Comment:
Is there a reason for this? I think this can be simplified:

```suggestion
        if (this.parent === null) {
            return "";
        }

        let parent = await Monitor.getParent(this.id);
        return parent.name;
```